### PR TITLE
Updated Japanese translation

### PR DESCRIPTION
**Pull request by KabanFriends**
I fixed few mistranslations/typos in Japanese translation file.
I also checked how it looks in the actual app (built by myself).

But... I found some strange things in the font...
For example, look at this screenshot:
image

Look at characters called Hiragana and Katakana (あいう and アイウ)
and Kanjis (設定)
They look like they are using different fonts. (Hiragana and Katakana(あいうアイウ) looks nice but Kanjis(設定) looks too "clean" like Times New Roman font)

So could you change the font, at least when it shows Japanese? ...Or add setting to select the font?

If you're going to change the font itself, I recommend using Meiryo font (It's included in Japanese Windows 10 (I don't know about english windows) and I think its not hard to find/get that font)

There's another problem because of Japanese incompatible font
image
Kanjis in this picture: 接続 使用 解放 必要
the first red marked texts (接続) are completely same character but lower one has weird italic effect, but it has no special formats

This should be fixed by using a new font
or at least please add a setting to select the font (maybe the list of installed fonts)

### CODE REVIEW

File: app/locale/ja.ts [hunk 1]
```diff
@@ -32,58 +32,58 @@ export default {
...
   networkTroubleshootingHelp: '<1>トラブルシューティングガイド</1> を確認してください。',
   noticeAddedMedia: '<Media id="{{mediaId}}">{{mediaTitle}}</Media> を追加しました。',
   noticeMediaError: '{{url}} をリクエスト中にエラーが発生しました。',
   noticeNowPlaying:
-    '再生中: <Media id="{{mediaId}}">{{mediaTitle}}</Media> requested by <Username id="{{userId}}">{{username}}</Username>.',
+    '再生中: <Media id="{{mediaId}}">{{mediaTitle}}</Media> (<Username id="{{userId}}">{{username}}</Username>のリクエスト)',
   noticeUserRequestJoin:
     '<Username id="{{userId}}">{{username}}</Username> が参加申請を送りました。',
   offline: 'オフライン',
   ok: 'OK',
```
Comment (KabanFriends) [inline app/locale/ja.ts]: Note: "OK" is used by Japanese people too

**Comment by KabanFriends** [comment]
image
some more pictures
this difference exists in official release version too
first 4 characters (ひらがな) are Hiraganas
last 2 are kanjis
you can clearly see pixels in kanji but its hard to see them in hiragana

**Comment by KabanFriends** [comment]
*misclosed*
im sorry

**Comment by AndreasWebdev** [comment]
@samuelmaddock 
I've been working on japanese content before, this was my softfallback font-family for japanese content (Just add the "normal" fonts before those)..

`font-family: 'Mplus 1p', 'Hiragino Sans', 'Hiragino Kaku Gothic Pro', '游ゴシック' , '游ゴシック体' , YuGothic , 'Yu Gothic', 'ＭＳ ゴシック' , 'MS Gothic', sans-serif;`

**Comment by samuelmaddock** [comment]
Really appreciate the effort @KabanFriends. The font used in Metastream is called [Karla](https://fonts.google.com/specimen/Karla). I'll see if I can make any improvements with fonts, otherwise I'll look into having a font preference based on language selection.

@AndreasWebdev good to know, I'll use that list if I can't fix up the Karla font.

**Comment by KabanFriends** [comment]
well there is an updated Japanese translation file that needs to be merged
you should merge it before this request will be forgotten

**Comment by KabanFriends** [comment]
ah misclose again

**Comment by KabanFriends** [comment]
there's another small problem but this one depends on the environment and only happens when you build it in unsupported environment (so this problem does NOT happen in already-built official releases)

image
left one: fixed language list
right one: broken language list 
The reason of that problem was flag codes, so removing flag code characters will fix the problem.

the index file itself was already broken since I cloned it into my computer (I added japanese in the list after cloning)
image

but I'm not sure those flag codes are compatible with japanese characters (like putting flag codes and japanese characters in the same file) because that problem started to happen in japanese environment.

So I recommend to remove flag codes and replace them with other things like "(US)" to prevent more problems to happen.

**Comment by samuelmaddock** [comment]
Going to merge the translation changes and track the font rendering issue in #83 83

**Pull request merged**